LayoutTests/imported/w3c/web-platform-tests/css/css-typed-om/the-stylepropertymap/properties/font-weight-expected.txt

```diff
@@ -9,7 +9,7 @@ PASS Can set 'font-weight' to the 'bold' keyword: bold
 PASS Can set 'font-weight' to the 'bolder' keyword: bolder
 PASS Can set 'font-weight' to the 'lighter' keyword: lighter
 PASS Can set 'font-weight' to a number: 0
-FAIL Can set 'font-weight' to a number: -3.14 assert_approx_equals: expected -3.14 +/- 0.000001 but got -3
+FAIL Can set 'font-weight' to a number: -3.14 assert_approx_equals: expected -3.14 +/- 0.000001 but got 0
```

Comment:
ditto

Reply:
Like above, font-weight is defined to [<number [1,1000]>](https://drafts.csswg.org/css-values-4/#number-value)], so the range check does not allow -3.